config/default.yml

```
@@ -4,6 +4,11 @@ Minitest:
     - '**/test/**/*'
     - '**/*_test.rb'
 
+Minitest/AnyAssertions:
+  Description: 'This cop checks for at least one assertion (or flunk) in tests.'
```

```suggestion
  Description: 'This cop checks for at least one assertion (or refutation) in tests.'
```
I think it's better to avoid the informal 'flunk' and stay consistent with minitest's language.

---

Flunk is a minitest method: https://www.rubydoc.info/github/seattlerb/minitest/Minitest%2FAssertions:flunk

I think it's subtly different to a refutation. A flunk, to me, means "this test hasn't been written yet". Maybe other people use it differently.

---

Ah, I hadn't come across that before.